dspace-server-webapp/src/main/java/org/dspace/app/rest/submit/step/validation/MetadataValidation.java

```diff
@@ -42,6 +44,10 @@ public class MetadataValidation extends AbstractValidation {
 
     private static final String ERROR_VALIDATION_REGEX = "error.validation.regex";
 
+    private static final String DOCUMENT_TYPE_FIELD =
```

Comment:
Same here, let's move this out of static field and just call the ConfigurationService where you need the value in methods below.

Reply:
Done!